### ISSUE TITLE
Charge - Authrorize

### ISSUE THREAD

**Issue by Akanowo** [Flutterwave/Flutterwave-node-v3#68]
I recently started getting the error below every time I try to authorize a payment.

"Cannot destructure property 'daily' of 'checkData(...)' as it is undefined."

The payload I'm sending is:
`{
 card_number: '5531886652142950',
 cvv: '564',
 expiry_month: '09',
 expiry_year: '32',
 currency: 'NGN',
 amount: 2028.4,
 email: 'akanowouko@gmail.com',
 fullname: 'Akanowo Uko',
 enckey: 'FLWSECK_TEST4a1163f79c09',
 tx_ref: 'FWL-65479001-364b-4a99-9e08-dafaeb990238',
 redirect_url: 'http://localhost:8080/api/v1/payment/verify',
 authorization: { mode: 'pin', pin: '3310' }
}`

The full error is
`{
 status: 'error',
 message: "Cannot destructure property 'daily' of 'checkData(...)' as it is undefined.",
 data: null
}`

**Comment by LinusMuema** [comment]
I am also facing the same issue.

My logs:
```bash
1|dev  | payload is  {
1|dev  |   currency: 'KES',
1|dev  |   email: 'linus.m.muema@gmail.com',
1|dev  |   amount: 550,
1|dev  |   tx_ref: '6PMuciWq',
1|dev  |   fullname: 'moose.m',
1|dev  |   meta: {
1|dev  |     data: '{"body":{"total":550,"bonus":31.872313007927357,"sold":"61f823dba04ea262d2b25c35","amount":256,"profit":113.50232179590542,"charges":[{"61e137fe8e7a9c234499b072":13},{"61e137fe8e7a9c234499b073":13},{"61e137fe8e7a9c234499b074":13},{"61e137fe8e7a9c234499b075":13},{"61e137fe8e7a9c234499b076":39},{"61e137fe8e7a9c234499b077":13},{"61e137fe8e7a9c234499b078":13},{"61e137fe8e7a9c234499b079":64}],"art":"61e138028e7a9c234499b0c4","buyer":"61e137fc8e7a9c234499b052","seller":"61e7cfeb2133681d8597056d","bought":"61fb94f0af1e6d4f81257a79"},"payment":"charge","type":"herzina"}'
1|dev  |   },
1|dev  |   phone_number: '254722000009'
1|dev  | }
```

and the result:
```bash
1|dev  | result is  {
1|dev  |   status: 'error',
1|dev  |   message: "Cannot destructure property 'daily' of 'checkData(...)' as it is undefined.",
1|dev  |   data: null
1|dev  | }
```

**Comment by Corvus97** [comment]
Hello @LinusMuema @Akanowo 👋🏽,

Thanks for sharing this with the team. Great news! this has been resolved.

Kindly retry the call now.

**Comment by Akanowo** [comment]
Thanks for the response, it has been resolved but another error occurred.

In verifying the transaction with the card otp, sometimes I get this response.

"upstream connect error or disconnect/reset before headers. reset reason: connection failure"

But then I get a webhook event response with the event type "charge.completed". And in the webhook too, verifying the transaction with the transaction id sometimes return the same error. Can that also be looked into cuz I've tested a couple of times and it happens but other times it goes through.
